docs/cas-server-documentation/configuration/Configuration-Properties.md

```diff
@@ -2493,6 +2493,9 @@ To fetch CRLs, the following options are available:
 # cas.authn.x509.refreshIntervalSeconds=3600
 # cas.authn.x509.maxPathLengthAllowUnspecified=false
 
+# SUBJECT_DN
```

Comment:
It would also be good to design a table in the docs preferably in this section to describe the meaning of each format string, similar to [this](https://apereo.github.io/cas/development/configuration/Configuration-Properties.html#health-endpoint)

Reply:
OK, I added a table with a description of each value.